datarobot_batch_scoring/batch_scoring.py

```diff
@@ -121,10 +122,18 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
         base_headers['content-type'] = 'text/csv; charset=utf8'
         if compression:
             base_headers['Content-Encoding'] = 'gzip'
+
         if import_id:
-            endpoint = base_url + '/'.join((import_id, 'predict'))
+            endpoint = base_url + import_id
+        else:
+            endpoint = base_url + '/'.join((pid, lid))
+
+        if max_prediction_explanations:
+            endpoint += '/reasonCodesPredictions?maxCodes=' + \
```

Comment:
this is baking in using the deprecated "reasonCodesPredictions" routes instead of the "predictionExplanations" routes.  Should we add a check that tried to see if predictionExplanations exists and uses that?

Reply:
Will change it.